### DISCUSSION TITLE
[MongoDB] Replication

### DISCUSSION THREAD

**Discussion by jojaeng2** [BDD-CLUB/challenge-every-week-posting#454] [1. 개발 관련 글 모음]
요즘 좀 바빠서 글을 많이 못쓰게 되네요
MongoDB의 복제 방식인 replicaset에 대해 정리하기 시작했습니다.
[블로그](https://blog.naver.com/ds4ouj/223420937840)

**Comment by 02ggang9** [comment]
쿠버 공부하면서 DB를 파드로 띄우면 어떻게 고가용성을 유지하지랑 로드밸런싱 문제 고민하고 있었는데 도움 많이 됐습니다. 감사합니다 bb
